### PR TITLE
Package functions for migration tests

### PR DESCRIPTION
**Pull request by mitiao**
For better reuse of migration functions
Verified runs:
online: http://147.2.207.208/tests/491
offline: http://147.2.207.208/tests/494
(ignore ltss and timeout in runs)

### CODE REVIEW

**Comment by coolo** [comment]
I'm not a native speaker either, but I have the impression that 'modulate' means something else :)

**Comment by mitiao** [comment]
How about package? :)

File: lib/migration.pm [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```
Comment (okurz) [inline lib/migration.pm]: Please use the reduced copyright notice, e.g. from tests/console/zypper_in.pm . I will prepare a PR to fix all other files

Comment (okurz) [inline lib/migration.pm]: oh wait, that is a different license. let me check.

Comment (okurz) [inline lib/migration.pm]: see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2772

Comment (okurz) [inline lib/migration.pm]: tests/console/yast_scc.pm is a better template

Comment (mitiao) [inline lib/migration.pm]: done

**Pull request merged**